llpc/test/shaderdb/relocatable_shaders/PipelineCs_PushConst.pipe

```diff
@@ -0,0 +1,93 @@
+; Test that the offset to the push constant area is correct when the push constants are spilled.
+; This may be a fragile test because the conditions in which the push constatns are spilled might change.
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s && llvm-objdump --arch=amdgcn --mcpu=gfx900 -d %t.elf | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: 0000000000000000 <_amdgpu_cs_main>:
+; This tries to match the sequece that loads the vector from the puch constant.  This is the only 2 element load in the
+; code.  The "4" is the offset of the push constant area in the root table.
+; SHADERTEST: s_mov_b32 [[r1:s[0-9]+]], 4 //
```

Comment:
nit: is the trailing `//` needed?

Reply:
Something is needed.  It hightlights the space after the 4.  Otherwise we could match a 48 in an different instruction.